developers_chamber/slack_utils.py

```diff
@@ -0,0 +1,49 @@
+import re
+
+from slack_sdk import WebClient
+
+from developers_chamber.qa.base import RepoMixin
+from developers_chamber.utils import run_command
+
+
+class RecentMigrationsSlackUploader(RepoMixin):
+
+    def __init__(self, token, channel, target_branch, migrations_pattern):
+        self.client = WebClient(token=token)
+        self.channel = channel
+        self.target_branch = target_branch
+        self.migrations_pattern = r'{}'.format(migrations_pattern)
+        self.active_branch = self._get_active_branch_name()
+
+    def _run_command(self, command):
```

Comment:
Maybe it doesn't seem but it is used. The class `RepoMixin' does not have definition of this method but it uses that. Check it [here](https://github.com/druids/developers-chamber/pull/105/files#diff-24ae4ab5827cf3265fe5ad9aafe34581f039ee7fb5f54f4fe08b294caf6e3259R89).

Reply:
I'd put something like this in `RepoMixin`:
```python
def _run_command(self, command):
    raise NotImplementedError()
```
so that the code isn't as magical (I'm looking at you, Celery).

Reply:
Yes it's definitely better. Done

Reply:
Why not just implement it in `RepoMixin`? There's not need to override.

Reply:
Of course, it works fine :) I changed it.